src/darwin/Framework/CHIP/MTROTAProviderDelegateBridge.mm

```diff
@@ -600,11 +631,12 @@ bool GetPeerNodeInfo(CommandHandler * commandHandler, const ConcreteCommandPath
 
                       handler->AddResponse(cachedCommandPath, response);
                       handle.Release();
+                      gOtaSender.ResetState();
                   }
 
-                                errorHandler:^(NSError *) {
-                                    // Not much we can do here
-                                }];
+                  errorHandler:^(NSError *) {
+                      gOtaSender.ResetState();
```

Comment:
Why?  This part does not make sense to me.  Not least the `assertChipStackLockedByCurrentThread` assert in `ResetState` will crash in this case.

Reply:
ah this shouldn't be here. will remove.

Reply:
done